docs/source/getting_started/install.md

```diff
@@ -11,17 +11,19 @@ required to run the pipeline.
 ??? example "Install for Python 3.8 and newer"
     Run in your terminal:
     ```shell
-    pip install mne-bids coloredlogs tqdm pandas scikit-learn json_tricks fire
+    pip install mne-bids autoreject coloredlogs tqdm pandas scikit-learn json_tricks fire
     ```
 
 ??? example "Install for older Python versions"
     Run in your terminal:
     ```shell
-    pip install mne-bids coloredlogs tqdm pandas json_tricks scikit-learn fire typing_extensions
+    pip install mne-bids autoreject coloredlogs tqdm pandas json_tricks scikit-learn fire typing_extensions
```

Comment:
we should have a requirements.txt file and suggest to use this.

Reply:
Will all be unnecessary once we start switching to `mne-bids-pipeline-runner`, which can hopefully happen within the next couple of days!